components/org.wso2.carbon.identity.application.authenticator.facebook/src/main/java/org/wso2/carbon/identity/application/authenticator/facebook/FacebookAuthenticator.java

```diff
@@ -351,7 +349,7 @@ protected void setSubject(AuthenticationContext context, Map<String, Object> jso
         return jsonObject;
     }
 
-    protected void buildClaims(AuthenticationContext context, Map<String, Object> jsonObject)
+    protected void buildClaims(AuthenticationContext context, Map<String, Object> jsonObject, ClaimConfig claimConfig)
```

Comment:
Shall we keep the old method and call the new method from the old method to preserve backward compatibility